Platform/Teams/Triage/Rules of Engagement with Triage.md

```diff
@@ -1,18 +1,36 @@
 # Preface
 This document will help all team members working on VA.gov understand when it is appropriate to engage the Triage team for support and provide relevant information to assist in routing reported issues to other appropriate teams.  Because of the limited number of resources on Triage, they cannot be a "catch-all" for every issue reported on the platform and every application.  Please read the following document carefully in order to determine the appropriate path for your incident.
 
-## When it is OK to engage the Triage Team
+## Is this is a Severity 1 Issue? - Notify On Call Immediatley
 
-* When the incident is believed to be **Severity 1** which can be defined as
-  * A significant amount of users are impacted
-  * Issue resolution is required in a 24 hour period
-  * Part of (or all of) va.gov is severely disrupted
+When the incident is believed to be **Severity 1** which can be defined as (but not limited to)
+ * Issue resolution is required in a 24 hour period
+ * Part of (or all of) va.gov is severely disrupted
   
-* When an issue is discovered that is not in your products scope of work
-  * Any issue that is discovered that is outside your teams scope of work can be reported to Triage
-  * In this case severity does not matter
+Examples:
+- va.gov is unresponsive
+- Platform errors causing extreme delays for users
```

Comment:
Another good example: 
```
- Login to va.gov is unavailable
```

Reply:
Also, I'm not sure that `- Platform errors causing extreme delays for users` is concrete enough. I'm a little fuzzy myself on exactly what that means.

Reply:
RE: Login being unavailable, would we want to discern from partial degradation, i.e. one of the providers is down (MHV) but not all?